plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -195,8 +195,17 @@ func (r *ResourceFit) PodFitsResources(pod *api.Pod, existingPods []*api.Pod, no
 		return false, err
 	}
 
-	if int64(len(existingPods))+1 > info.Status.Capacity.Pods().Value() {
-		glog.V(10).Infof("Cannot schedule Pod %+v, because Node %+v is full, running %v out of %v Pods.", podName(pod), node, len(existingPods), info.Status.Capacity.Pods().Value())
+	var allocatable api.ResourceList
+
+	// In case old version kubelet does not set Allocatable
```

Comment:
This shouldn't be necessary. This is already performed when converting to the internal API format, [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/defaults.go#L202).